package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "share-code-creator",
-  "version": "2.1.2",
-  "description": "",
+  "version": "2.1.3",
```

Comment:
It is recommended to not change the version in pull requests.

If the version is not changed in the pull request, the module author can bundle multiple changes together for a single release.